source/exec.tex

```diff
@@ -5672,3 +5687,424 @@
 return as_awaitable(std::forward<Value>(value), static_cast<Promise&>(*this));
 \end{codeblock}
 \end{itemdescr}
+
+\rSec1[exec.par.scheduler]{Parallel scheduler}
+
+\pnum
+\tcode{parallel_scheduler} models \libconcept{scheduler}.
+
+\pnum
+Let \tcode{sch} be an object of type \tcode{parallel_scheduler}, and
+let \tcode{\exposid{BACKEND-OF}(sch)} be \tcode{*ptr},
+where \tcode{sch} is associated with \tcode{ptr}.
+
+\pnum
+The expression \tcode{get_forward_progress_guarantee(sch)} returns
+\tcode{forward_progress_guarantee::paral\-lel}.
+
+\pnum
+Let \tcode{sch2} be an object of type \tcode{parallel_scheduler}.
+Two objects \tcode{sch} and \tcode{sch2} compare equal if and only if
+\tcode{\exposid{BACKEND-OF(sch)}} and
+\tcode{\exposid{BACKEND-OF(sch2)}} refer to the same object.
+
+\pnum
+Let \tcode{rcvr} be a receiver.
+A \defn{proxy} for \tcode{rcvr} with base \tcode{B} is
+an lvalue \tcode{r} of type \tcode{B} such that
+\begin{itemize}
+\item
+\tcode{r.set_value()} has effects equivalent to
+\tcode{set_value(std::move(rcvr))}.
```

Comment:
The paper does not add the periods in the first two bullets, so ideally, those would be fixup commits.

Reply:
I'm not bothering to highlight small punctuation changes in separate commits.